apps/news-site/news-site-next/src/components/dialog/dialog.jsx

```diff
@@ -1,21 +1,16 @@
-import { useState, useEffect } from "react";
 import classNames from "classnames";
 
 import Toggle from "../toggle/toggle";
 import { useDataContext } from "@/context/data-context";
+import { useLocalStorage } from "@/hooks/use-local-storage";
 
 import styles from "news-site-css/dist/dialog.module.css";
 
 export default function Dialog({ onClose }) {
-    const [reduceMotion, setReduceMotion] = useState(false);
-    const [highContrast, setHighContrast] = useState(false);
+    const [reduceMotion, setReduceMotion] = useLocalStorage("news-site-settings-reduced-motion", false);
+    const [highContrast, setHighContrast] = useLocalStorage("news-site-settings-high-contrast", false);
```

Comment:
This might be nice to be configurable in the future? (e.g. juse Cookies, LocalStorage, or the indexDB) but that is independent of this PR.